### ISSUE TITLE
Reproducing Table 1 (sphere/prism, Appendix A.8.1/A.8.2): image_combination disabled in predict.py, and missing training hyperparameters

### ISSUE THREAD

**Issue by Athenaxr** [yunfei-teng/CryoGEN#1]
Hi, thanks for releasing the code and for a really interesting paper. I'm trying to reproduce the sphere/prism rows of Table 1 using the released code and the description in Appendix A.8.1/A.8.2, and I'm hitting a specific, reproducible gap I'd appreciate your help with.

**What I did**

- Generated the sphere (140³, hollow shell, outer diameter 70, thickness 4, wedge ±60°, 10 random-origin 96³ subtomograms) and prism (96³, side 60, 10 rotations) exactly as described in A.8.1/A.8.2.
- Trained with bin/cryogen.py train, batch size 1 (per A.7), lr 4e-4 (default), noise_level 0.2 (default), swept checkpoints across the full training run.
- Predicted with bin/cryogen.py predict, cube_size 64, crop_size 96.

**The specific issue**

models/tools.py has an image_combination() function that projects the measured Fourier region back onto the prediction (a data-consistency step). It's imported in bin/predict.py but the call is commented out:
# predicted = image_combination(noisy_img=in_data, pred_img=predicted)

**Questions**

1. Was image_combination() applied when producing the sphere/prism numbers in Table 1? If so, was it done inside predict.py, or as a separate post-processing step not in the current release?
2. If it's intentionally disabled by default, why, and is there a flag or recommended way to turn it on for reproducing Table 1?
3. For the sphere/prism experiments specifically: how many training epochs / optimizer steps / save_model_epochs were used to obtain the Table 1 checkpoint? A.7 gives batch size (1) and learning rate (4e-4) but no epoch/iteration count. Knowing the actual epoch count used would remove a lot of guesswork.
4. A.8.1 doesn't specify the sphere's foreground/background grey values or whether the shell edge is anti-aliased (Fig. 13(a) shows what looks like a ~1-voxel soft edge). Since PSNR uses max(GT) and SSIM's constants scale with it, this materially changes absolute PSNR/SSIM even when the relative accuracy is identical. Could you confirm the exact values used?
5. A.8.2 states prism thickness = 20 pixels, but measuring Fig. 13(b) directly (calibrated against the sphere's 70-pixel diameter in Fig. 13(a), which measures exactly 70.00) gives a z-extent of ≈26.5 pixels. Is 20 the pre-resampling value, or is there a rotation/resampling step not mentioned in A.8.2 that would account for the difference?

Happy to share our generation/training/eval scripts if useful for comparison. Thanks again for the work!
